### PR TITLE
Feature add bank statement

### PR DESCRIPTION
**Pull request by sk057**

### CODE REVIEW

File: Controllers/FileManager.cs [hunk 1]
```diff
@@ -35,9 +34,18 @@ public void Import()
...
+                    bool quotes = false;
+                    if(line[0] == '\"')
+                    {
+                        quotes = true;
+                    }
+                    ReadSEB(reader, db, line, quotes);
+                }
+                else if (line.Substring(0, 24) == "Operacijos/Balanso Tipas" || line.Substring(0, 16) == "Transaction type")
+                {
+                    ReadLum(reader, db, line);
```
Comment (AleksandrasMaliuginas) [inline Controllers/FileManager.cs:48]: For better readablity could be writtn in `swich case` style & put in a separate method (named, for example, MachCsvFormat or MachHeader)

**Review by AleksandrasMaliuginas — APPROVED** [review]
Maybe all logic could be simplified using some CSV library (eg. CSVHelper).
I will talk about it today, nevertheless **please, commit your changes _immediately_ after merging** : )

File: Controllers/FileManager.cs (hunk 1 above)
Comment (sk057) [reply Controllers/FileManager.cs:48]: I will look into it

**Review by imila254 — APPROVED** [review]
Looks great

**Pull request closed without merging**